libs/designsystem/src/lib/components/progress-circle/progress-circle.component.spec.ts

```diff
@@ -43,24 +44,44 @@ describe('ProgressCircleComponent', () => {
     });
 
     describe('diameter', () => {
-      it('should default to md (56px)', () => {
-        spectator.detectChanges();
-        expect(spectator.component.diameter).toBe(ProgressCircleComponent.DIAMETER_MAP.md);
+      it('should default to md', () => {
+        expect(spectator.component.diameter).toBe(56);
       });
-      it('should map sm to 40px', () => {
+
+      it('should map sm to correct value', () => {
         spectator.setInput({ size: 'sm' });
-        spectator.detectChanges();
-        expect(spectator.component.diameter).toBe(ProgressCircleComponent.DIAMETER_MAP.sm);
+        expect(spectator.component.diameter).toBe(40);
       });
-      it('should map md to 56px', () => {
+
+      it('should map md to correct value', () => {
         spectator.setInput({ size: 'md' });
-        spectator.detectChanges();
-        expect(spectator.component.diameter).toBe(ProgressCircleComponent.DIAMETER_MAP.md);
+        expect(spectator.component.diameter).toBe(56);
       });
-      it('should map lg to 96px', () => {
+
+      it('should map lg to correct value', () => {
         spectator.setInput({ size: 'lg' });
-        spectator.detectChanges();
-        expect(spectator.component.diameter).toBe(ProgressCircleComponent.DIAMETER_MAP.lg);
+        expect(spectator.component.diameter).toBe(96);
+      });
+    });
+
+    describe('strokeWidth', () => {
+      it('should default to md', () => {
+        expect(spectator.component.strokeWidth).toBe(4);
```

Comment:
Consider changing to
```suggestion
        expect(spectator.component.strokeWidth).toBe('md');
```